CONTRIBUTING.rst

```diff
@@ -93,6 +93,23 @@ Contribution Guidelines
 .. _formatting: https://molecule.readthedocs.io/en/latest/testing.html#formatting
 .. _linting: https://molecule.readthedocs.io/en/latest/testing.html#linting
 
+Code Of Conduct
+===============
+
+Please see our `Code of Conduct`_ document.
+
+.. _Code of Conduct: ./.github/CODE_OF_CONDUCT.md
```

Comment:
This file is sourced from docs: https://github.com/ansible/molecule/blob/e012419defe06daceb1ed2bc8a4f64996a45fbaa/docs/contributing.rst#L3

Which probably means that we cannot use relative references like this here because in the rendered RTD website this will point to a non-existing location (unless we also wire and publish that page there).